python-sdk/src/astro/files/base.py

```diff
@@ -69,6 +70,21 @@ def is_binary(self) -> bool:
         result: bool = self.type.name == constants.FileType.PARQUET
         return result
 
+    def is_pattern(self) -> bool:
+        """
```

Comment:
@sunank200 Can we add a test case for this?

Reply:
Also, if we can validate that this works with the glob pattern would be nice, since we will be working on that going forward.